### PR TITLE
Rosettacode: add 3 new task.

### PR DESCRIPTION
**Pull request by ventilooo**
Task:
* Bull and Cow
* Guess the number
* Rock Paper Scissor

Waiting comment to fix (it should need a review) @Morriar and @R4PaSs

### CODE REVIEW

**Comment by privat** [comment]
test this please

File: examples/rosettacode/guess_the_number.nit [hunk 1]
```diff
@@ -15,4 +15,5 @@ loop
 		print "Well guessed"
 		break
 	end
+if stdin.eof then break
```
Comment (R4PaSs) [inline examples/rosettacode/guess_the_number.nit]: Indentation is off

File: examples/rosettacode/rock_paper_scissor.nit [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	if player == "rock" and computer == "scissors" then return 1
+	if player == "rock" and computer == "paper" then return 0
+	if player == "paper" and computer == "rock" then return 1
+	if player == "paper" and computer == "scissors" then return 0
+	if player == "scissors" and computer == "paper" then return 1
+	return 0
+end
+
+# Return a String for the player about what the computer player did.
+fun computer_message(str: String): String do
```
Comment (R4PaSs) [inline examples/rosettacode/rock_paper_scissor.nit]: I'm not sure about the use of this one, a simple string concatenation could very well do the job

File: examples/rosettacode/rock_paper_scissor.nit [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	if str == "rock" then return "The computer played rock"
+	if str == "scissors" then return "The computer played scissors"
+	return "The computer played paper"
+end
+
+# Return a random value for the computer player input.
+fun computer_player : String do
```
Comment (R4PaSs) [inline examples/rosettacode/rock_paper_scissor.nit]: Same for this function, the code could just be a part of the main

File: examples/rosettacode/rock_paper_scissor.nit [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	var tab = ["rock", "scissors", "paper"]
+	return tab.rand
+end
+
+# Game loop.
+var player_input = ""
+var tab = ["rock", "scissors", "paper"]
+var tab_message = ["You lose", "You win!", "It's a tie!"]
+loop
```
Comment (R4PaSs) [inline examples/rosettacode/rock_paper_scissor.nit]: Handle the `stdin.eof` here too

**Comment by R4PaSs** [comment]
There's no "Bull and Cow" module in the PR, why is it in the covering letter ?

**Comment by R4PaSs** [comment]
The commit chain is ugly too, please rebase

**Comment by privat** [comment]
ping?

**Comment by ventilooo** [comment]
Actually correcting the code and as the missing bulls and cow files. 
However, I'm trying to figure out how to correctly handle stdin.eof. 
I don't understand clearly what it does and do include it in my code.

**Pull request closed without merging**